### DISCUSSION TITLE
How do we add the undo/redo buttons in the Toolbar?

### DISCUSSION THREAD

**Discussion by marchrenno** [sibiraj-s/ngx-editor#403] [Q&A] [answered]
I have added the {history: true} option when creating the instance of Editor. But the Undo and Redo button are not showing in the toolbar. I tried adding them in my instance of Toolbar, using naively 'undo', 'redo', but it doesn't seem to recognize the words. Is there any specific import or modification that needs to be done?

The shortcuts Ctrl+y and Ctrl+z seems to be working though to undo and redo changes.

Thank you.

**Comment by sibiraj-s** [comment]
Yes. they are not available in menubar yet. Will check this.

**Reply by fabbemannen** [reply]
Hello! Has there been an update to this?

**Reply by sibiraj-s** [reply, marked as answer]
Undo and redo is now available in menubar. Checkout https://sibiraj-s.github.io/ngx-editor/menu/
